zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,214 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+            return err, ret
+        except shell.ShellError:
+            return True, None
+
+def _writeSysfs(path, value, supressRaise=False):
+    try:
+        with open(path, 'w') as f:
+            f.write(str(value))
+    except Exception as e:
+        logger.warn(str(e))
+        if not supressRaise:
+            raise OvsError(str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)